src/flask_pyoidc/flask_pyoidc.py

```diff
@@ -163,7 +163,10 @@ def _handle_authentication_response(self):
         logger.debug('received authentication response: %s', authn_resp.to_json())
 
         try:
-            result = AuthResponseHandler(client).process_auth_response(authn_resp, auth_request)
+            if "OIDC_CLOCK_SKEW" in current_app.config:
```

Comment:
* With above suggestion of `process_auth_response` accepting a dict, this could be simplified:
   ```python
   extra_token_args = {}
   if "OIDC_CLOCK_SKEW" in current_app.config:
       extra_token_args['skew'] = current_app.config['OIDC_CLOCK_SKEW']
   result = AuthResponseHandler(client).process_auth_response(authn_resp, auth_request, extra_token_args)
   ```
* This code base uses single quotes from strings, so it should be `'OIDC_CLOCK_SKEW'`
* This new config option should be documented [here](https://github.com/zamzterz/Flask-pyoidc/blob/main/docs/configuration.md#flask-configuration) and with a new test added for it.